samples/cellular/gnss/sample.yaml

```diff
@@ -13,3 +13,105 @@ tests:
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
     tags: ci_build sysbuild
+  # Following configurations will be used by the positioning CI integration
+  # job to verify PRs
+  sample.cellular.gnss.integration_config_positioning_agnss_nrfcloud_ltem_pvt:
+    sysbuild: true
+    build_only: true
+    extra_configs:
+      - CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
+      - CONFIG_LTE_NETWORK_MODE_NBIOT_GPS=n
```

Comment:
I would remove those network modes that are 'n'. This is a choice symbol so you can only have one of them on anyway.

Reply:
We configure all gnss sample images like this for extra clarity, so that it's obvious what the image does, and what not. I'd rather use the same format for these images, so that it's easy to see that the configurations match.

Reply:
OK, fine. I understand this in many symbols but for choice symbols it may also be confusing because setting them all 'y' doesn't really do that and you don't necessarily know which one is used.
But I'll leave this to you.

Reply:
For testers it's not always obvious which kconfigs are choice and which are not. You need to search it in source to understand how it works after it is passed on to the compiler. For me it is better to state clearly that this image _is LTE-M_ and _is not NB-IoT_.
We pass all these y/n parameters to test code as text vectors, and generate required variants automatically. It's better to keep the list of y's and n's unmodified, so that we know that images build in place A have exactly the same configuration as images built in place B.

Reply:
> For testers it's not always obvious which kconfigs are choice and which are not. You need to search it in source to understand how it works after it is passed on to the compiler.

Same applies for developers. You only know it by looking at the KConfig from the source code or documentation.

> For me it is better to state clearly that this image _is LTE-M_ and _is not NB-IoT_.

I understand this but there is a danger of misconfiguration as well if you set both to 'y'. You don't basically know what will be chosen and you don't get compilation errors or warnings.

> We pass all these y/n parameters to test code as text vectors, and generate required variants automatically. It's better to keep the list of y's and n's unmodified, so that we know that images build in place A have exactly the same configuration as images built in place B.

I don't necessarily see how this would change (but I don't know how these text vectors are implemented in detail) if you would have just one line of Kconfig for a choice symbol but yes, you need to know that it's choice symbol by looking at the documentation. But this is something you should be aware of anyway to avoid setting multiple values for it.

I do understand the value of setting other (non-choice symbols) configurations even if it was a default.